low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -127,7 +103,11 @@ def validate(self):
         pass
 
     def process(self):
-        Cmd._call_script(f"{self._script_dir}/{self.script}", self._args)
+        setup_sspl = f"{self._script_dir}/{self.script} {' '.join(self._args)}"
+        output, error, returncode = SimpleProcess(setup_sspl).run()
+        if returncode != 0:
+            sys.stderr.write("error: %s\n\n" % str(error))
+            sys.exit(errno.EINVAL)
```

Comment:
Return SetupError() exception from there main function should be printing and returning. Example....
```
def main(argv: dict):
    try:
...
        command = Cmd.get_command(desc, argv[1:])
        command.process()

    except Exception as e:
        sys.stderr.write("error: %s\n\n" % str(e))
        sys.stderr.write("%s\n" % traceback.format_exc())
        Cmd.usage(argv[0])
        return errno.EINVAL


if __name__ == '__main__':
    sys.exit(main(sys.argv))
```